Update README files (#14)
canonical/http-proxy-operators: squid-forward-proxy-operator/README.md

# Squid forward proxy charm

[![CharmHub Badge](https://charmhub.io/squid-forward-policy/badge.svg)](https://charmhub.io/squid-forward-policy)
[![Publish to edge](https://github.com/canonical/http-proxy-operators/actions/workflows/publish.yaml/badge.svg)](https://github.com/canonical/http-proxy-operators/actions/workflows/publish.yaml/badge.svg)
[![Promote charm](https://github.com/canonical/http-proxy-operators/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/http-proxy-operators/actions/workflows/promote_charm.yaml)
[![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)

> Squid forward proxy charm is a charm running Squid proxy for its forward HTTP 
> proxy functionality.

Reviewer: ```suggestion
The Squid forward proxy charm is a charm running a Squid proxy to forward HTTP 
proxies.
```
"for its forward HTTP proxy functionality" is a bit confusing to me, so I tried to simplify the sentence. If you want to emphasize the functionality aspect, then an alternative suggestion is below (although I think this suggestion is a bit clunky to read).
```suggestion
The Squid forward proxy charm is a charm running a Squid proxy for its functionality to 
forward HTTP proxies.
```

Author: Here, "forward" is actually an adjective used to describe the HTTP proxy server. This emphasizes that it is a *forward* HTTP proxy server, as opposed to a *reverse* proxy server. You can see the [definition here](https://en.wikipedia.org/wiki/Proxy_server#Forward_proxy_vs._reverse_proxy)

Maybe it should be called "HTTP forward proxy" instead of "forward HTTP proxy" to make it clearer?

Reviewer: Thank you for clarifying! I agree with you that "HTTP forward proxy" is clearer

Author: Updated to "HTTP forward proxy", thanks!

Like any Juju charm, this charm supports one-line deployment, configuration, 
integration, scaling, and more. For Charmed Squid forward proxy, this includes:

* Automatically installing and setting up Squid as a forward HTTP proxy
* HTTP proxy access control via Juju relations

For information about how to deploy, integrate, and manage this charm, see the
official [Squid forward proxy charm documentation](https://charmhub.io/squid-forward-proxy).

## Get started

In this section, we will deploy the Squid forward proxy charm and use it as an
HTTP proxy server.

You’ll need a workstation, such as a laptop, with sufficient resources to launch
a virtual machine with 4 CPUs, 8 GB RAM, and 50 GB of disk space.

### Set up

> You can follow the tutorial [here](https://documentation.ubuntu.com/juju/3.6/howto/manage-your-deployment/manage-your-deployment-environment/#set-things-up) to set up a test environment for Juju with LXD.

Reviewer: ```suggestion
You can follow the how-to guide [here](https://documentation.ubuntu.com/juju/3.6/howto/manage-your-deployment/manage-your-deployment-environment/#set-things-up) to set up a test environment for Juju with LXD.
```

### Deploy

From inside the virtual machine, deploy the Squid forward proxy charm using the
`juju deploy` command.

```
juju deploy squid-forward-proxy --channel latest/edge
```

### Basic operations

When the Squid forward proxy charm has finished deploying and installing, it 

> should show the message `ready: 0`.

Reviewer: I assume this message shows when you run `juju status`? It might be good to clarify

Author: Added, thanks!
